agent/api/task.go

```diff
@@ -248,7 +248,7 @@ func (task *Task) UpdateMountPoints(cont *Container, vols map[string]string) {
 // there was no change
 // Invariant: task known status is the minimum of container known status
 func (task *Task) updateTaskKnownStatus() (newStatus TaskStatus) {
-	seelog.Debug("Updating task: %s", task.String())
+	seelog.Debugf("Updating task: %s", task.String())
```

Comment:
I'll leave that for a later time.  This log line is badly placed anyway; the method here doesn't actually update it, it just returns the new status if one is possible (or `TaskStatusNone` when it shouldn't change).